libs/platform/form/src/renderers/default-form.renderer.ts

```diff
@@ -191,14 +247,10 @@ export class DefaultFormRenderer implements FormRenderer {
       <oryx-checkbox
         .required=${field.required}
         .style=${this.resolveStyles(field)}
+        ?hasError=${field.attributes?.hasError}
       >
-        <input
-          type="checkbox"
-          .name=${field.id}
-          ?checked=${!!value}
-          ?required=${field.required}
-        />
-        ${field.label}
+        ${this.renderInput(field, value)}
+        <span>${field.label}</span>
```

Comment:
Can be a breaking change.

Reply:
Added in BC description

Reply:
This is not how we are dealing with breaking changes :) Added a general comment https://github.com/spryker/oryx/pull/753#issuecomment-1723353836